source/extensions/filters/http/alternate_protocols_cache/filter.cc

```diff
@@ -38,6 +38,7 @@ Filter::Filter(const FilterConfigSharedPtr& config, Event::Dispatcher& dispatche
     : cache_(config->getAlternateProtocolCache(dispatcher)), time_source_(config->timeSource()) {}
 
 Http::FilterHeadersStatus Filter::encodeHeaders(Http::ResponseHeaderMap& headers, bool) {
+
```

Comment:
nit: Remove this?

Reply:
Given I've got another PR in this space and github is broken I'll do in my follow up :-)